app/views/my_facilities/bp_not_controlled.html.erb

```diff
@@ -167,6 +167,30 @@
             </tbody>
           </table>
         <% end %>
+        <div class="d-flex mt-4">
+          <em class="h-12px w-12px mr-8px bg-red br-6px"></em>
+          <p class="fs-12px lh-1 c-grey-dark">
+            &gt;30%
+          </p>
+        </div>
+        <div class="d-flex">
+          <em class="h-12px w-12px mr-8px bg-orange br-6px"></em>
+          <p class="fs-12px lh-1 c-grey-dark">
+            21-30%
+          </p>
+        </div>
+        <div class="d-flex">
+          <em class="h-12px w-12px mr-8px bg-yellow br-6px"></em>
+          <p class="fs-12px lh-1 c-grey-dark">
+            11-20%
+          </p>
+        </div>
+        <div class="d-flex">
+          <em class="h-12px w-12px mr-8px b-2px bg-green-medium br-6px"></em>
+          <p class="fs-12px lh-1 c-grey-dark">
+            0-10%
+          </p>
+        </div>
```

Comment:
The `BP not controlled` and `Missed visits` pill key doesn't match the CSS in `_dashboard.css`.
❌ `0%` is grey in CSS, but isn't listed as a key item
❌ `1-10%` should be green
✔️ `11-20%` should be yellow
✔️ `21-30%` should be orange
❌ `≥31%` should be red

Reply:
Done